stage1/files/nftables.conf

```
@@ -0,0 +1,56 @@
+#!/usr/sbin/nft -f
+
+flush ruleset
+
+table inet filter {
+	chain input {
+		type filter hook input priority 0
+		policy drop # Drop this.
+
+		ct state { established, related } counter accept
+		ct state invalid drop
+
+		iif lo accept
+
+		ip protocol icmp accept
+		meta l4proto icmpv6 accept
+
+		tcp dport ssh accept comment "Allow SSH on v4 and v6"
+		tcp dport {http, https} accept comment "Allow HTTP on v4 and v6"
+		tcp dport 9001 accept comment "Allow MQTT over WebSockets"
+
+		udp dport dns accept comment "Allow DNS"
+		udp dport dhcp accept comment "Allow DHCP"
```

```suggestion
		udp dport bootps accept comment "Allow DHCP"
```
This should be recognised.